community_topics_workflow.md

```diff
@@ -8,61 +8,71 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
-- [ ] Adds the topic to the [Board](https://github.com/orgs/ansible-community/projects/2/views/1).
+- [ ] A Committee person checks the topic's content, asks the author/other persons to provide additional information if needed.
 
 ### Discussion stage
 
 - [ ] If the topic is ready to be discussed, the Committee person:
 
-  - [ ] Adds the `discussion_topic` label.
-  - [ ] Moves the topic to the `In Discussion` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+  - [ ] Adds the `community-meeting` tag if it needs to be discussed in the meeting.
   - [ ] Opens the discussion by adding a comment asking the Community and the Committee to take part in it.
 - [ ] No synchronous discussion is needed (there are no blockers, complications, confusion, or impasses).
 
 ### Voting stage
 
 - [ ] Depending on the topic's complexity, 1-2 weeks after the discussion was opened, the Committee person formulates vote options based on the prior discussion and gives participants reasonable amount of time to propose changes to the options (no longer than a week). The person summarizes the options in a comment and also establishes a date when the vote begins if there are no objections about the options / vote date.
 - [ ] In the vote date, the vote starts with the comment of a Committee person which opens the vote and establishes a date when the vote ends ($CURRENT_DATE + no longer than 21 days; usually it should not exceed 14 days, 21 days should only be used if it is known that a lot of interested persons will likely not have time to vote in a 14 days period).
-- [ ] The Committee person labels the topic with the `active-vote` label and moves the topic to the `Active Vote` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+- [ ] The Committee person labels the topic with the `active-vote` tag.
 - [ ] The Committee person adds `[Vote ends on $YYYY-MM-DD]` to the beginning of the topic's description.
+- A vote is actually two polls, one for the Steering Committee, one for everyone else. To create a vote in a topic:
+  - [ ] Create a new post in the topic.
+  - [ ] Click the `gear` button in the composer and select Build Poll.
+  - [ ] Click the `gear` in the Poll Builder for advanced mode.
+  - [ ] Set up the options (generally this will be Single Choice but other poll types can be used).
+  - [ ] Title it "Steering Committee vote" and "Limit voting" to the `Steering Committee`.
+  - [ ] Do not set the close date because this cannot be changed later.
+  - [ ] Results should be "Always Visible" unless there is some good reason for the SC votes not to be public.
+  - [ ] Submit the poll (the BBcode will appear in the post) and then repeat the above for the second poll.
+    - [ ] Title should be "Community vote".
+    - [ ] No group limitation.
+    - [ ] Results "Staff only" (i.e. an anonymous vote, but admins can check if there are concerns).
```

Comment:
Should we keep all votes like they were before we adopted the forum?

Reply:
@gotmax23 i'm not sure i understand though I'm for keeping them as they were. Please suggest the wording change

Reply:
Sorry, I was missing a word. I meant keep all votes public instead of making community votes anonymous.

Reply:
@gotmax23 ah, yeah, i would prefer to see them public too.
@GregSutcliffe do the formulations above imply community votes to be private? Is there any rationale for that and can we just remove the point containing "Staff only"?